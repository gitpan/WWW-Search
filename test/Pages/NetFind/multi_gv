http://netfind.aol.com/dirsearch.adp?query=ferfer+it
http://netfind.aol.com/web.adp?query=ferfer%20it&first=16&last=30
