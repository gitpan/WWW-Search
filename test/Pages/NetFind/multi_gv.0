
<HTML>
<HEAD>
  <TITLE>AOL Search Preview : Search Results for ferfer it
</TITLE>
  <LINK REL=stylesheet TYPE=text/css HREF=/gr/styles.css>

</HEAD>
<BODY BGCOLOR="#ffffff" TEXT="#000000" LINK="#0000cc" VLINK="#000066" ALINK=#000099>
<!--- BEGIN NAV BAR ---> 
<CENTER>
<TABLE WIDTH=585 CELLPADDING=0 CELLSPACING=0 BORDER=0 ALIGN=CENTER>
        <TR>
                <TD WIDTH=119 BGCOLOR="#FFFFFF" ALIGN="LEFT"><A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/"><IMG SRC="http://www.aol.com/gr/mega_aol_banner.gif" WIDTH=119 HEIGHT=26 BORDER=0 ALIGN="LEFT"></A></TD>
                <TD WIDTH=466 BGCOLOR="#CC0000" VALIGN=MIDDLE ALIGN="CENTER"><NOBR>&nbsp;<FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/netfind/" STYLE="COLOR:#FFFFFF"><FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Search</B></FONT></A>&nbsp;|&nbsp;<A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/webcenters/" STYLE="COLOR:#FFFFFF"><FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Web Centers</B></FONT></A>&nbsp;|&nbsp;<A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/shopping/" STYLE="COLOR:#FFFFFF"><FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Shopping</B></FONT></A>&nbsp;|&nbsp;<A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/community/" STYLE="COLOR:#FFFFFF"><FONT COLOR="#FFFFFF" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Community</B></FONT></A>&nbsp;|&nbsp;<A HREF="http://dynamic.aol.com/cgi/redir?http://www.aol.com/downloadaol.adp" STYLE="COLOR:#FFCC66"><FONT COLOR="#FFCC66" STYLE="font-family:sans-serif, Arial, Helvetica; font-size: 13px; font-weight:bold"><B>Download AOL</B></FONT></A></FONT>&nbsp;</NOBR></TD>
        </TR>
        <TR>
                <TD WIDTH=585 HEIGHT=12 COLSPAN=2><HR SIZE=1></TD>
        </TR>
        <TR>
                <TD WIDTH=585 HEIGHT=12 COLSPAN=2>You are here: <A HREF=http://dynamic.aol.com/cgi/redir?http://www.aol.com/home.html>Home</A> &gt; AOL.COM Search</TD>
        </TR>
</TABLE>
</CENTER>

<!--- END NAV BAR --->

<P ALIGN=Center>
<A HREF=http://ads.web.aol.com/link/2014570/aol><img alt="Check Out Our Advertiser! ClickHere." border=0 height=60 width=468 src="http://ads.web.aol.com/image/2014570/aol?search=ferfer%20it"></A>

<P ALIGN=Center>
<CENTER>
  <TABLE WIDTH=585 BGCOLOR=#000066 BORDER="0" CELLSPACING="0" CELLPADDING="3"
  ALIGN="Center">
    <TR>
      <TD ROWSPAN=3 WIDTH=200 ALIGN=CENTER VALIGN=TOP><A HREF="index.adp"><IMG SRC="/gr/logo_com.gif"

	    WIDTH="198" HEIGHT="48" BORDER="0"></A></TD>
      <TD WIDTH=385><FONT COLOR=#ffffff>Enter search words below and then click
	Search.</FONT></TD>
    </TR>
<FORM ACTION="dirsearch.adp">
    <TR>
      <TD>
	  <TABLE BORDER="0" CELLSPACING="0" CELLPADDING="0">
	    <TR>
	      <TD>
		<INPUT TYPE=text NAME=query SIZE=25 VALUE="ferfer it">
</TD>
	      <TD>
		<IMG SRC=/gr/pixel.gif WIDTH=3 HEIGHT=10 BORDER=0><INPUT TYPE=submit VALUE="Search!"></TD>
	    </TR>
	  </TABLE>
      </TD>
    </TR>
	</FORM>
    <TR>
      <TD><A HREF="index.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Home</FONT></A>
	<FONT COLOR="#ffffff">|</FONT>


	<A HREF="refine.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Search
	Options</FONT></A> <FONT COLOR="#ffffff">|</FONT>
	<A HREF="/help/index.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Help</FONT></A></TD>
    </TR>
  </TABLE>
</CENTER>
<CENTER>
  <TABLE WIDTH=585 BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR> 
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
      <TD WIDTH=565 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="10"
	    BORDER="0"></TD>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
    </TR>
  </TABLE>
</CENTER>
<CENTER>
  <TABLE WIDTH=585 BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
      <TD WIDTH=440 VALIGN=TOP>
<TABLE CELLPADDING=4 CELLSPACING=0 BORDER=0>
<TR>
<TD>
<!-- RESULTS -->
</DIV>
<P>
<P>
<TABLE CELLPADDING=0 CELLSPACING=0 WIDTH=430 BORDER=0>
<TR>
<TD WIDTH=200><FONT COLOR=#990000 STYLE="font-family: Arial, helv, helvetica;  font-weight:bold; font-size:14px">MATCHING WEB PAGES</FONT></TD><TD WIDTH=50>&nbsp;</TD><TD WIDTH=130 ALIGN=CENTER>(<B>1</B> thru <B>15</B> of <B>19</B>)</TD><TD WIDTH=50 ALIGN=CENTER><A HREF="web.adp?query=ferfer%20it&first=16&last=30"><FONT STYLE="font-family: Arial, helv, helvetica;  font-weight:bold; font-size:12px">next</FONT></A> <B>&gt;&gt;</B></TD></TR></TABLE>
<DIV ALIGN=left STYLE="font-size:10px; font-weight:light">
Some of the following results are from the World Wide Web and may contain objectionable material that AOL does not endorse.</DIV>
<P>		
<DIV STYLE="font-weight:normal">		<FONT COLOR=#990000 STYLE="font-weight:bold"><B>99%</B></FONT> <A HREF="http://addistribune.ethiopiaonline.net/Archives/1997/11/28-11-97/pm269.htm " STYLE="font-weight:bold"><B>
		Prime Minister Visits Flood Victims in Somali Region </B></A><BR>Prime Minister Visits Flood Victims in Somali Region On Friday, 21 November, Prime Minister Meles Zenawi visited flood victims in Somali Region of Ethiopia. He pledged the Federal government will do everything in supplying assistance  including.. <BR><I>http://addistribune.ethiopiaonline.net/Archiv</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>98%</B></FONT> <A HREF="http://www.healthnet.org/programs/promed-hma/9802/msg00145.html " STYLE="font-weight:bold"><B>
		PRO/AH> Rift Valley fever - Ethiopia </B></A><BR>[Date Prev][Date Next][Thread Prev][Thread Next][Date Index][Thread Index] PRO/AH> Rift Valley fever - Ethiopia To: promed-ahead@usa.healthnet.org Subject: PRO/AH> Rift Valley fever - Ethiopia From: ProMED-mail <promed@usa.healthnet.org> Date:  Sun. <BR><I>http://www.healthnet.org/programs/promed-hma/</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>98%</B></FONT> <A HREF="http://www.comesa.int/comms/commrotl.htm " STYLE="font-weight:bold"><B>
		COMESA List of Road Transit Corridors </B></A><BR>COMESA Transport&Communications List of Road Transit Corridors (a) MAPUTO CORRIDOR (i) Maputo - Big Bend - Lavumisa; (ii) Maputo - RSA border - Johannesburg; (iii) Maputo - Chicualacuala - border with Zimbabwe - Harare ; and (iv) Maputo -  Inhambane <BR><I>http://www.comesa.int/comms/commrotl.htm </I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>98%</B></FONT> <A HREF="http://cgi.biwa.or.jp/~t-tamoto/wacky/race10/out02.htm " STYLE="font-weight:bold"><B>
		http://cgi.biwa.or.jp/~t-tamoto/wacky/race10/...</B></A><BR>Ê NO CarNO TIME CarName Driver Player Group I b d 101 1249 4919.29 X{gSS-999 Ryu g´kyutech 59.86 102 1450 4923.78 ú{¸Ó[Ó[62.37 103 1793 4926.26 hey.baby!takihiro takihito 77.74 104 1816 4935.71 b L±¤²¡²G 78.79 105 1807 4960.11 IX-F981 SHIGE  IX-55.. <BR><I>http://cgi.biwa.or.jp/~t-tamoto/wacky/race10/</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>97%</B></FONT> <A HREF="http://www.sas.upenn.edu/African_Studies/eue_web/95/r5floods.htm " STYLE="font-weight:bold"><B>
		UNDP EMERGENCIES UNIT FOR ETHIOPIA </B></A><BR>Main Menu UNITED NATIONS DEVELOPMENT PROGRAMME Emergencies Unit for Ethiopia Update on Response to Floods of Gode Zone, Region 5 UN-Emergencies Unit for Ethiopia, 05 June 1995 Background Unusually severe flooding of the Shabelle River toward  the.. <BR><I>http://www.sas.upenn.edu/African_Studies/eue_</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>95%</B></FONT> <A HREF="http://www.lainet.com/~initaly/msgboard/maillist.htm " STYLE="font-weight:bold"><B>
		People Looking For People Message Board Index </B></A><BR>[Main Page] [Message Board] Post your message. Italian citizenship by birthright From: "Carol Markino" <carolmarkino@hotmail.com> Venice Calendars for 2000 From: "bah" <bah@gateway.net> No Subject From: "Olivetta Balestra" <olie@ozemail.com.au>  I.. <BR><I>http://www.lainet.com/~initaly/msgboard/maill</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>94%</B></FONT> <A HREF="http://homepages.go.com/~charbell/bidi02.htm " STYLE="font-weight:bold"><B>
		BidiN1 </B></A><BR>IL-PRASPAR TAL-BIDI BADU - 1998 By Charles J. Belli ©Copyright 1998 Shortcut to Jokes | 1| |2| |3| |4| |5| |6| |7| |8| |9| |10| |11| |12| |13| |14| BACK .1997 Jokes .1999 Jokes 1 Il-Praspar tal-Bidi Badu Back to the Top Dan l-ahhar, il-Bidi  Badu.. <BR><I>http://homepages.go.com/~charbell/bidi02.htm </I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>94%</B></FONT> <A HREF="http://primarycare.medscape.com/other/ProMED/1998/02.98/ProMED.v98.n047.html " STYLE="font-weight:bold"><B>
		ProMED 23 Feb 98; Vol. 98, No. 047 </B></A><BR>Medscape Home Site Map Marketplace My Medscape CME Center Feedback Help Desk ProMED Digest Monday, 23 February 1998 Volume 98 : Number 047 In this issue: PRO/AH: Blue tongue - Kenya (04) PRO/AH/EDR: Rift Valley fever - Uganda PRO/AH: Rift  Valley.. <BR><I>http://primarycare.medscape.com/other/ProMED/</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>94%</B></FONT> <A HREF="http://id.medscape.com/other/ProMED/1998/02.98/ProMED.v98.n047.html " STYLE="font-weight:bold"><B>
		ProMED 23 Feb 98; Vol. 98, No. 047 </B></A><BR>Medscape Home Site Map Marketplace My Medscape CME Center Feedback Help Desk ProMED Digest Monday, 23 February 1998 Volume 98 : Number 047 In this issue: PRO/AH: Blue tongue - Kenya (04) PRO/AH/EDR: Rift Valley fever - Uganda PRO/AH: Rift  Valley.. <BR><I>http://id.medscape.com/other/ProMED/1998/02.9</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>93%</B></FONT> <A HREF="http://www.ethemb.se/news_nov97.htm " STYLE="font-weight:bold"><B>
		Ethiopian Embassy Information </B></A><BR>NOVEMBER NEWS 1997 November 26 Negasso calls for further Ethio-Italian dev't cooperation Speaker confers with IPPF President Women empowerment said best approach to peace building process Defence Force members assist flood victims November  25.. <BR><I>http://www.ethemb.se/news_nov97.htm </I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>93%</B></FONT> <A HREF="http://edcintl.cr.usgs.gov/adds/c1/r2/et/doc/admn/etadmn4.html " STYLE="font-weight:bold"><B>
		Ethiopia - Administrative Level 4 </B></A><BR>Ethiopia - Administrative Level 4 Metadata: Identification_Information Data_Quality_Information Spatial_Data_Organization_Information Spatial_Reference_Information Entity_and_Attribute_Information Distribution_Information  Metadata_Reference.. <BR><I>http://edcintl.cr.usgs.gov/adds/c1/r2/et/doc/</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>90%</B></FONT> <A HREF="http://www.infolink.com.br/~peco/LIGONG.HTM " STYLE="font-weight:bold"><B>
		ONGs no Pensamento Ecológico </B></A><BR>LIGATIONE a atualizado em: 06/04/1999 pelo: Pensamento Ecológico http://www.infolink.com.br/~peco/index.htm Inicio em Setembro de 1996 Relação de entidades/organizações: Se você conhece alguma que não esteja nesta relação mande as  infomações.. <BR><I>http://www.infolink.com.br/~peco/LIGONG.HTM </I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>81%</B></FONT> <A HREF="http://www.initaly.com/~initaly/msgboard/maillist.htm " STYLE="font-weight:bold"><B>
		People Looking For People Message Board Index </B></A><BR>[Main Page] [Message Board] Post your message. Nut festivals From: "BESANA UK" <besanauk@btintenet.com> house swap during Sydney Olympics From: " Peter E Power" <powerpe@ozemail.com.au> Information on Campobasso and Avellino From: "JO-ANN KEADY"  . <BR><I>http://www.initaly.com/~initaly/msgboard/mail</I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>78%</B></FONT> <A HREF="http://disc.server.com/Indices/24209.html " STYLE="font-weight:bold"><B>
		DALnet #chatzone Board </B></A><BR>This is the #chatzone DALnet discussion forum. This forum is provided as an oportunity for any member of the #chatzone community to compose perhaps more lengthy and lucid comments or information than is sometimes possible amongst the hustle  and.. <BR><I>http://disc.server.com/Indices/24209.html </I>
		<P><FONT COLOR=#990000 STYLE="font-weight:bold"><B>74%</B></FONT> <A HREF="http://www.ionet.net/~jjbryan/looney.html " STYLE="font-weight:bold"><B>
		Looney Page </B></A><BR>My Looney Page This is a page I have dedicated to Looney MUD and all the crazy people who mud there. For those of you who are new to this page, Looney is a MUD. A MUD is a multi-user dungeon. It is a lot of fun, BUT it can be quite addicting.  For.. <BR><I>http://www.ionet.net/~jjbryan/looney.html </I>
		<P></DIV><P>

<!-- END RESULTS -->
</TD>
</TR>
</TABLE>
</TD>
     
<TD WIDTH=2 BGCOLOR=#000066><IMG SRC=/gr/pixel.gif WIDTH=2 HEIGHT=1 BORDER=0></TD>
<TD WIDTH=123 VALIGN=TOP BGCOLOR=#000066>
<!-- SIDEBAR -->

<TABLE HEIGHT=100% WIDTH=100% BORDER=0 CELLSPACING=0 CELLPADDING=0>
   <TR>
    <TD VALIGN=TOP><TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0>
	<TR>
	  <TD><IMG SRC=/gr/bdot.gif WIDTH=123 HEIGHT=1 BORDER=0></TD>
	</TR>
	<TR>
	  <TD>
<TABLE CELLPADDING=6 CELLSPACING=0 WIDTH=100% BORDER=0 BGCOLOR=#ccffcc>
<TR>
<TD ALIGN=CENTER><B>Search books:</B><BR>
<B>
<A HREF="http://dynamic.aol.com/cgi/redir?http://www.amazon.com/exec/obidos/aol-search/search=ferfer%20it">ferfer it</B><BR>
<A HREF="http://dynamic.aol.com/cgi/redir?http://www.amazon.com/exec/obidos/aol-search/search=ferfer%20it"><IMG SRC=/gr/amazonblk88x31.gif WIDTH=88 HEIGHT=31 BORDER=0></A>
</TD>
</TR>
</TABLE>

	  </TD>
	</TR>
<TR><TD><IMG SRC=/gr/pixel.gif WIDTH=1 HEIGHT=2 BORDER=0></TD></TR>  <TR>
    <TD>
</TD>
	</TR>
	<TR>
	  <TD><IMG SRC=/gr/bdot.gif WIDTH=123 HEIGHT=1 BORDER=0></TD>
	</TR>

	<TR>
	  <TD><IMG SRC=/gr/bdot.gif WIDTH=123 HEIGHT=1 BORDER=0></TD>
	</TR>
	<TR>
	  <TD><br>
<FONT COLOR=#ffffff><b>AOL Instant Messenger&#153;</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/aim/home.html"><FONT COLOR=#ffff00 CLASS=yellow>Instant Messenger&#153 2.0</FONT></a> is your free instant link to your friends, family.</FONT>
<p>


<FONT COLOR=#ffffff><b>Shop@AOL</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://shop.aol.com/"><FONT COLOR=#ffff00 CLASS=yellow>Great Bargains:</FONT></a> Digital shopping, right at your fingertips, from Shop@AOL.</FONT>
<p>


<FONT COLOR=#ffffff><b>Download AOL Now!</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/tryaol/"><FONT COLOR=#ffff00 CLASS=yellow>Try AOL:</FONT></a>. Free download. Try AOL now, and get 100 free hours, too.</FONT>
<p>


<FONT COLOR=#ffffff><b>Read Your News</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/mynews/home.adp"><FONT COLOR=#ffff00 CLASS=yellow>My News:</FONT></a> Get your news, from latest headlines to sports scores -all from My News.</FONT>
<p>


<FONT COLOR=#ffffff><b>Back to School '99</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/back2school/"><FONT COLOR=#ffff00 CLASS=yellow>School Days:</FONT></a> Chat with other students, get homework help, check out the latest fashions and more.</FONT>
<p>


<FONT COLOR=#ffffff><b>Web Centers</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/webcenters/realestate/home.adp"><FONT COLOR=#ffff00 CLASS=yellow>Real Estate Web Center:</FONT></a> From mortgages to rentals to moving, get all the answers to your real estate questions.</FONT>
<p>


<FONT COLOR=#ffffff><b>Your Sports</b><br>
<a href="http://dynamic.aol.com/cgi/redir?http://www.aol.com/webcenters/sports/home.adp"><FONT COLOR=#ffff00 CLASS=yellow>Sports Web Center:</FONT></a> Catch up on the latest sports news, team reports, scores, and more.</FONT>
<p>






</TD>
	</TR>
      </TABLE>
    </TD>
  </TR>
	<TR>
	  <TD><IMG SRC=/gr/bdot.gif WIDTH=123 HEIGHT=1 BORDER=0></TD>
	</TR>
  <TR>
    <TD VALIGN=BOTTOM ALIGN=CENTER>
<TABLE HEIGHT=100%>
<TR VALIGN=BOTTOM>
<TD VALIGN=BOTTOM>
      <A HREF="http://www.dmoz.org"><IMG SRC="/gr/dmoz.gif" WIDTH="88" HEIGHT="31"
	  BORDER="0"></A>
</TD>
</TR>
</TABLE>
</TD>
  </TR>
</TABLE>
<!-- END SIDEBAR -->
</TD>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
    </TR>
  </TABLE>
</CENTER>
<CENTER>
<TABLE WIDTH=585 CELLPADDING=0 CELLSPACING=0 BORDER=0>
<TR>
<TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC=/gr/pixel.gif width=10 height=0 border=0></TD>
<TD WIDTH=565>
  <TABLE WIDTH=565 BGCOLOR=#000066 BORDER="0" CELLSPACING="0" CELLPADDING="3"
  ALIGN="Center">
    <TR>

      <TD ROWSPAN=3 WIDTH=200 ALIGN=CENTER VALIGN=TOP><A HREF="index.adp"><IMG SRC="/gr/logo_com.gif"

	    WIDTH="198" HEIGHT="48" BORDER="0"></A></TD>
      <TD WIDTH=385><FONT COLOR=#ffffff>Enter search words below and then click
	Search.</FONT></TD>
    </TR>
<FORM ACTION="dirsearch.adp">
    <TR>
      <TD>
	  <TABLE BORDER="0" CELLSPACING="0" CELLPADDING="0">
	    <TR>
	      <TD>
		<INPUT TYPE=text NAME=query SIZE=25 VALUE="ferfer it">
</TD>
	      <TD>
		<IMG SRC=/gr/pixel.gif WIDTH=3 HEIGHT=10 BORDER=0><INPUT TYPE=submit VALUE="Search!"></TD>
	    </TR>
	  </TABLE>
      </TD>
    </TR>
	</FORM>
    <TR>
      <TD><A HREF="index.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Home</FONT></A>
	<FONT COLOR="#ffffff">|</FONT>
	<A HREF="refine.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Search
	Options</FONT></A> <FONT COLOR="#ffffff">|</FONT>
	<A HREF="/help/index.adp"><FONT COLOR=#ffff00 STYLE="font-size:14px;font-weight:bold">Help</FONT></A></TD>
    </TR>
  </TABLE>
</TD>
<TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC=/gr/pixel.gif width=10 height=0 border=0></TD>
</TR>
</TABLE>
</CENTER>

<!-- BOTTOM AD -->
<!--
<CENTER>
  <TABLE WIDTH=585 BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
      <TD WIDTH=565 ALIGN=CENTER>
<BR>
<A HREF=http://ads.web.aol.com/link/2014570/aol><img alt="Check Out Our Advertiser! ClickHere." border=0 height=60 width=468 src="http://ads.web.aol.com/image/2014570/aol?search=ferfer%20it"></A>

<BR>&nbsp;
</TD>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
    </TR>
  </TABLE>
</CENTER>
 -->
<!-- /END BOTTOM AD -->

<CENTER>
  <TABLE WIDTH=585 BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
      <TD WIDTH=565><!-- ALSO SEARCH IN -->
<CENTER>
  <TABLE WIDTH=565 BORDER="0" CELLSPACING="3" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD><FONT COLOR="#003366"><B>Also Search In:</B></FONT></TD>
    </TR>
    <TR>
      <TD ALIGN=CENTER>
<A HREF="web.adp?query=ferfer%20it">Web Articles</A>
<B>&#149;</B>
<A HREF="http://hometown.aol.com/find.adp?cid=10000&resultOrder=1&queryToFlush=&query=ferfer%20it">Personal Home Pages</A> <B>&#149;</B>
      <A HREF="http://www.download.com/pc/list/1,339,0-a-0-0-1,00.html?tag=st.dl.fd.srch.list&queryType=quicksearch&search=ferfer%20it">Downloads</A>
      <B>&#149;</B>
      <A HREF="http://comptonsv3.web.aol.com/search/articlesearch?db1=A&db2=P&db3=F&db4=S&perpage=25&view=viewcomptons&userstring=ferfer%20it">Encyclopedias</A>
      <B>&#149;</B>
      <A HREF="http://www.deja.com/=aol/dnquery.xp?Search=Find%21&query=ferfer%20it">Newsgroups</A>
      <B>&#149;</B>
      <A HREF="http://www.thriveonline.com/cgi-bin/thriveall?vcs_personality=thrive%5fall&vcs_mode=search%5fdb&vcs_range=1%2d60&thriveterm=ferfer%20it">Health</A>

</TD>
    </TR>
    <TR>
      <TD><FONT COLOR="#003366"><B>Other Searches:</B></FONT></TD>
    </TR>
    <TR>
      <TD ALIGN=CENTER>
<A HREF="http://classifiedplus.aol.com/">ClassifiedPlus</A>
	<B>&#149;</B> <A HREF="http://www.moviefone.com/">Movie Times</A>
	<B>&#149;</B> <A HREF="http://www.aol.com/netfind/whitepages.adp">White
	Pages</A> <B>&#149;</B>
	<A HREF="http://www.aol.com/netfind/yellowpages.adp">Yellow Pages</A>

</TD>
    </TR>
  </TABLE>
</CENTER>
<!-- END ALSO SEARCH -->
<IMG SRC=/gr/pixel.gif WIDTH=565 HEIGHT=1 BORDER=0>
      </TD>
      <TD WIDTH=10 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="10" HEIGHT="1"
	    BORDER="0"></TD>
    </TR>
    <TR>
      <TD COLSPAN=3 BGCOLOR=#0066cc><IMG SRC="/gr/pixel.gif" WIDTH="585" HEIGHT="10"
	    BORDER="0"></TD>
    </TR>
  </TABLE>
</CENTER>
<P>
<CENTER>
  <TABLE BORDER="0" CELLSPACING="0" CELLPADDING="0" ALIGN="Center">
    <TR>
      <TD ALIGN=CENTER><FONT STYLE="font-size:12px"><A HREF="/link_to_us.adp">Link to Us</A> | <A HREF="/about.adp">About AOL.COM
	Search</A> | <A HREF="/add.adp">Add Your Site</A> |
	<A HREF=http://mediaspace.web.aol.com>Advertise with Us</A><BR>Questions? Comments? <A HREF="/feedback.adp">Send us feedback.</A></FONT></TD>
    </TR>
    <TR>
      <TD><IMG SRC="/gr/pixel.gif" WIDTH="1" HEIGHT="5" BORDER="0"></TD>
    </TR>
    <TR>
      <TD ALIGN=CENTER><FONT STYLE="font-size:10px">Copyright &copy; 1999 America Online, Inc.<BR>
	All rights reserved. <A HREF="/copyright.adp">Legal Notices</A><BR>
<A HREF="http://www.aol.com/tryaol">Try AOL</A><BR>
<A HREF="http://www.aol.com/info/privacy.html">Privacy Policy</A></FONT></TD>
    </TR>
  </TABLE>
</CENTER>


</BODY></HTML>
