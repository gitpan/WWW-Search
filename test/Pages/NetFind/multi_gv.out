


http://addistribune.ethiopiaonline.net/Archives/1997/11/28-11-97/pm269.htm 
http://www.healthnet.org/programs/promed-hma/9802/msg00145.html 
http://www.comesa.int/comms/commrotl.htm 
http://cgi.biwa.or.jp/~t-tamoto/wacky/race10/out02.htm 
http://www.sas.upenn.edu/African_Studies/eue_web/95/r5floods.htm 
http://www.lainet.com/~initaly/msgboard/maillist.htm 
http://homepages.go.com/~charbell/bidi02.htm 
http://primarycare.medscape.com/other/ProMED/1998/02.98/ProMED.v98.n047.html 
http://id.medscape.com/other/ProMED/1998/02.98/ProMED.v98.n047.html 
http://www.ethemb.se/news_nov97.htm 
http://edcintl.cr.usgs.gov/adds/c1/r2/et/doc/admn/etadmn4.html 
http://www.infolink.com.br/~peco/LIGONG.HTM 
http://www.initaly.com/~initaly/msgboard/maillist.htm 
http://disc.server.com/Indices/24209.html 
http://www.ionet.net/~jjbryan/looney.html 
http://www.math.okstate.edu/~bryjenn/looney.html 
http://www.webquarry.com/~catfish/guestbook.html 
http://www.azstarnet.com/~balstad/jenbook.htm 
http://www.irdu.nus.sg/silkair/guestbook5.html 
